tests/ClientBuilderTest.php

```diff
@@ -246,6 +248,26 @@ public function optionsDataProvider(): array
         ];
     }
 
+    public function testSdkIdentifierAndVersionDefaultValues(): void
```

Comment:
Can we rename this test to `testClientBuilderFallbacksToDefaultSdkIdentifierAndVersion` please? Also it seems to me that there is a missing test that assert that those values can be overwritten using the setters